src/main/java/selic/re/discordbridge/DiscordBotImpl.java

```diff
@@ -378,24 +379,22 @@ private void broadcastNoMirror(Text message) {
             server.execute(() -> this.broadcastNoMirror(message));
             return;
         }
-        MessageType type = MessageType.CHAT;
-        UUID sender = new UUID(0, 0);
 
-        server.sendSystemMessage(message, sender);
+        server.sendMessage(message);
 
         for (ServerPlayerEntity serverPlayerEntity : server.getPlayerManager().getPlayerList()) {
-            serverPlayerEntity.sendMessage(message, type, sender);
```

Comment:
Has this change been tested?

Reply:
Yes.

The "sender" field doesn't exist there anymore (and we had it nulled-out anyway). The MessageType was changed to SYSTEM because otherwise the game now posts a message as an empty user, so messages would look like `<> <DiscordUserName> message` with an extra set of `<>`